lib/routes/run.js

```diff
@@ -1,20 +1,63 @@
 'use strict';
 var chalk = require('chalk');
+var findUp = require('find-up');
+var inquirer = require('inquirer');
 var namespaceToName = require('yeoman-environment').namespaceToName;
+var path = require('path');
+var userHome = require('user-home');
+
+// Checks if .yo-rc.json exists in parent directory
+var filePath = findUp.sync('.yo-rc.json');
```

Comment:
I wonder if this check should be in the run function. I want to say it should be. Closer to the code that uses it and runs when it is actually needed instead of every time yo is started up